wdtk-datamodel/src/main/java/org/wikidata/wdtk/datamodel/implementation/StatementImpl.java

```diff
@@ -74,6 +74,36 @@
 
 	private List<SnakGroup> qualifiersGroups;
 
+	/**
+	 * Checks if the statement ID consists of a first part
+	 * with the entity id of the item the statement belongs to, the separator $, plus
+	 * a random hash of the form
+	 * /^\{?[A-Z\d]{8}-[A-Z\d]{4}-[A-Z\d]{4}-[A-Z\d]{4}-[A-Z\d]{12}\}?\z/
+	 *
+	 * @param statementID
+	 * @return true if the statement ID is of the valid format
+	 */
+	public static boolean validateStatementID(String statementID)
```

Comment:
If you raise an exception if the ID is not valid, then returning a boolean is not useful. If the function returns, then you aready know the ID is valid.